### PR TITLE
Decentralized deployment

### PR DESCRIPTION
**Pull request by koredefashokun**
## Description
This PR makes changes to the Kwenta build config and routing structure, that make it possible to deploy the site to IPFS.

## Related issue
N/A

## Motivation and Context
This is an important step in decentralizing the hosting of the Kwenta interface.

## How Has This Been Tested?
N/A

### CODE REVIEW

**Review by platschi — DISMISSED** [review]
All issues reported on Discord have been resolved, works flawless so far, couldn't find any more bugs for now. Well done!

File: pages/dashboard/history.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import DashboardLayout from 'sections/dashboard/DashboardLayout';
+import History from 'sections/dashboard/History';
+
+type HistoryPageProps = React.FC & { getLayout: (page: HTMLElement) => JSX.Element };
+
+const HistoryPage: HistoryPageProps = () => {
+	return <History />;
```
Comment (Tburm) [inline pages/dashboard/history.tsx:7]: We should add a title to these pages:
image

File: pages/dashboard/markets.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import DashboardLayout from 'sections/dashboard/DashboardLayout';
+import MarketsPage from 'sections/dashboard/Markets';
+
+type MarketsProps = React.FC & { getLayout: (page: HTMLElement) => JSX.Element };
+
+const Markets: MarketsProps = () => {
+	return <MarketsPage />;
```
Comment (Tburm) [inline pages/dashboard/markets.tsx:7]: We should add a title to these pages:
image

File: sections/shared/Layout/AppLayout/Header/BalanceActions.tsx [hunk 3]
```diff
@@ -32,40 +32,46 @@ const BalanceActions: FC = () => {
 
 	const sUSDBalance = synthBalances?.balancesMap?.[Synths.sUSD]?.balance ?? zeroBN;
 
-	const OPTIONS = useMemo(() => {
-		const accessiblePositions =
-			futuresPositions?.filter((position) => position.remainingMargin.gt(zeroBN)) ?? [];
+	const accessiblePositions = useMemo(
+		() => futuresPositions?.filter((position) => position.remainingMargin.gt(zeroBN)) ?? [],
```
Comment (Tburm) [inline sections/shared/Layout/AppLayout/Header/BalanceActions.tsx:36]: `futuresPositions` is empty if you visit the markets page first. If you switch to dashboard and back, it is complete. This results in the `BalanceActions` to be empty if a user visits the markets page before the dashboard.

**Review by Tburm — APPROVED** [review]

**Review by platschi — APPROVED** [review]

**Pull request merged**